spec/lib/search_spec.rb

```diff
@@ -2637,10 +2637,14 @@ def indexed_post(*args)
 
     it "allows to define custom filter" do
       expect(Search.new("advanced").execute.posts).to eq([post1, post0])
+
       Search.advanced_filter(/^min_chars:(\d+)$/) do |posts, match|
         posts.where("(SELECT LENGTH(p2.raw) FROM posts p2 WHERE p2.id = posts.id) >= ?", match.to_i)
       end
+
       expect(Search.new("advanced min_chars:50").execute.posts).to eq([post0])
+    ensure
```

Comment:
Usually in RSpec to clean things after an example, an `after` block is used. We can also use `Hash#clear` to simplify how to handle all this. So replacing the two `ensure` blocks with:
```ruby
after do
  Search.advanced_filters.clear
  Search.advanced_orders.clear
end
```

Reply:
Yeah, not sure what's better 🤔 

1/ Doing a `.clear` after every test, incurring a small cost, but we don't need to think about it anymore
2/ Manually have to remember to clear the "custom filter/order" when using it in a test

Reply:
The less we have to think about it, the better I’d say :grin: